lms/djangoapps/certificates/apis/v0/tests/test_views.py

```diff
@@ -65,29 +106,115 @@ def test_permissions(self):
         self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
         self.assertEqual(resp.data, {u'detail': u'Not found.'})  # pylint: disable=no-member
         self.client.logout()
 
         # same student of the certificate
-        self.client.login(username=self.student.username, password='test')
+        self.client.login(username=self.student.username, password=USER_PASSWORD)
         resp = self.client.get(self.get_url(self.student.username))
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
         self.client.logout()
 
         # staff user
-        self.client.login(username=self.staff_user.username, password='test')
+        self.client.login(username=self.staff_user.username, password=USER_PASSWORD)
         resp = self.client.get(self.get_url(self.student.username))
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
 
+    def test_inactive_user_access(self):
+        """
+        Verify inactive users - those who have not verified their email addresses -
+        are allowed to access the endpoint.
+        """
+        self.client.login(username=self.student.username, password=USER_PASSWORD)
+
+        self.student.is_active = False
+        self.student.save()
+
+        resp = self.client.get(self.get_url(self.student.username))
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+
+    def test_dot_valid_accesstoken(self):
+        """
+        Verify access with a valid Django Oauth Toolkit access token.
+        """
+        # make sure the client is not logged in
+        self.client.logout()
+        resp = self.client.get(
+            self.get_url(self.student.username),
+            HTTP_AUTHORIZATION="Bearer {0}".format(self.dot_access_token)
+        )
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+
+    def test_dot_invalid_accesstoken(self):
+        """
+        Verify access with an invalid Django Oauth Toolkit access token.
+        """
+        # make sure the client is not logged in
+        self.client.logout()
+        resp = self.client.get(
+            self.get_url(self.student.username),
+            HTTP_AUTHORIZATION="Bearer fooooooooooToken"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    def test_dot_expired_accesstoken(self):
+        """
+        Verify access with an expired Django Oauth Toolkit access token.
+        """
+        # make sure the client is not logged in
+        self.client.logout()
+        # set the expiration date in the past
+        self.dot_access_token.expires = datetime.utcnow() - timedelta(weeks=1)
+        self.dot_access_token.save()
+        resp = self.client.get(
+            self.get_url(self.student.username),
+            HTTP_AUTHORIZATION="Bearer {0}".format(self.dot_access_token)
+        )
+        self.assertEqual(resp.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    def test_dot_refresh_accesstoken(self):
```

Comment:
There isn't much value to this test. You've confirmed the endpoint works with a valid access token. At this point, dealing with refresh tokens is doing more to test Django OAuth Toolkit than the endpoint itself.